### ISSUE TITLE
Time slider bug

### ISSUE THREAD

**Issue by alixunderplatz** [tmrowco/electricitymap-contrib#2293]
Hi there,
it seems that moving the time slider doesn't change the displayed values in the tooltip anymore.
To reproduce, try the following:

App:
1.) select a country
2.) tap on one of the bars to show the tooltip box
3.) move the slider
--> the bars size will change, but the data in the tooltip stays the same

Browser:
1.) select a country
2.) select the time slider
3.) move your arrow to one of the bars/graphs
4.) use left and right arrows to move slider
--> the bars size will change, but the data in the tooltip stays the same

**Comment by fbarl** [comment]
Good catch @alixunderplatz!

This was probably introduced in one of my recent React migration PRs (maybe #2276), I'll take a look at it.

**Comment by fbarl** [comment]
As I wasn't aware the keyboard navigation was possible, I made #2276 in such a way that only mouse events trigger toolbar updates.

So some non-trivial changes are necessary to observe time slider changes for tooltip position updates (mainly, we need to base the position on other variables than the current cursor position). These changes might be easier to introduce if we skip storing tooltip state in Redux altogether so I would like to try addressing it as a part of https://github.com/tmrowco/electricitymap-contrib/issues/2288.

FYI @alixunderplatz, I was able to reproduce your steps using the _right_ arrow, but left arrow does nothing both in my Chrome and Firefox. I'm curious if _left_ arrow update the time slider for you and if yes, which browser are you using? :)
